doc/src/input/systems/interactions/potentials.rst

```diff
@@ -207,3 +207,30 @@ and ``B`` (well width) should be provided as strings.
 
 .. caution::
     ``B`` has to be positive
+
+
+Mie potential
+-------------
+
+The Mie potential is a classical potential for pair interactions
+expressed as:
+
+.. math::
+
+    V(r) = \frac{n}{n-m} \left(\frac{n}{m}\right)^{m/(n-m)}\epsilon
+    \left[ \left( \frac{\sigma}{r}\right)^n - \left( \frac{\sigma}{r}\right)^m \right]
+
+The potential type keyword is ``mie`` and the parameters are
+``sigma`` (:math:`\sigma`), the particle diameter, and ``epsilon`` (:math:`\epsilon`), the energetic
+parameter, which should be provided as strings (with units).
+The repulsive exponent ``n`` and the attractive exponent ``m`` should be provided as
```

Comment:
Just a side question: are m and n often used as non integers ?

Reply:
In general you can use them as additional parameters for forcefield optimization. I've seen repulsive exponents that were floats, it's not too common though. The attactive exponent is mostly left as 6 which is physically motivated.

Reply:
Ok, thanks!